### PR TITLE
Reversing labs a1000v2 action update

### PR DESCRIPTION
**Pull request by ivukovicRL**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added the localOnly parameter to get_classification action - when True action retrieves only local classification data for the sample, without falling back to querying TitaniumCloud.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by ivukovicRL** [comment]
Hi @JasBeilin We have an error on cirleci tests. 

`Possible backwards compatibility break, Your updates to this file contains changes to a name or an argument of an existing command(s).
Please undo you changes to the following command(s):
reversinglabs-a1000-get-classification`

It is not really a breaking change it is only a new parameter extracted. Action was using default value for this parameter before. Any advice on how we can fix this?

**Review by Noy-Maimon — CHANGES_REQUESTED** [review]
Thank you for your contribution! Looks great.
Please note my comment.

File: Packs/ReversingLabs_A1000/Integrations/ReversingLabsA1000v2/ReversingLabsA1000v2.yml [hunk 1]
```diff
@@ -263,6 +263,14 @@ script:
       name: hash
       required: true
       secret: false
+    - default: false
+      defaultValue: 'False'
+      description: Return only local classification data for the sample, without falling
+        back to querying TitaniumCloud.
+      isArray: false
+      name: localOnly
+      required: true
```
Comment (Noy-Maimon) [inline Packs/ReversingLabs_A1000/Integrations/ReversingLabsA1000v2/ReversingLabsA1000v2.yml:272]:
```suggestion
      required: false
```

**Comment by Noy-Maimon** [comment]
> Hi @JasBeilin We have an error on cirleci tests.
> 
> `Possible backwards compatibility break, Your updates to this file contains changes to a name or an argument of an existing command(s). Please undo you changes to the following command(s): reversinglabs-a1000-get-classification`
> 
> It is not really a breaking change it is only a new parameter extracted. Action was using default value for this parameter before. Any advice on how we can fix this?

Hi, this is due to the new argument being set as mandatory (required: true).
I added a comment for fixing that, and that should fix the circleCI failure.

File: Packs/ReversingLabs_A1000/ReleaseNotes/2_0_4.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### ReversingLabs A1000 v2
+Added the *localOnly* parameter to the **reversinglabs-a1000-get-classification** command that allows retrieving only local classification data for the sample, without falling back to querying TitaniumCloud.
```
Comment (Noy-Maimon) [inline Packs/ReversingLabs_A1000/ReleaseNotes/2_0_4.md:3]: @amshamah419  Hi, can you please verify this RN? thanks

**Review by Noy-Maimon — APPROVED** [review]

**Comment by ivukovicRL** [comment]
@Noy-Maimon Thanks for help :)

**Comment by Noy-Maimon** [comment]
> @Noy-Maimon Thanks for help :)

You're welcome! thank you for your contribution.

**Pull request merged**